static/js/settings_org.js

```diff
@@ -418,104 +418,177 @@ function _set_up() {
         settings_ui.disable_sub_setting_onchange(this.checked, "id_realm_message_content_edit_limit_minutes", true);
     });
 
-    exports.save_organization_settings = function () {
-        _.each(property_types.settings, function (v, k) {
-            property_type_status_element(k).hide();
-        });
+    function property_value_element_refers(property_name) {
+        if (property_name === 'realm_message_content_edit_limit_minutes') {
+            return Math.ceil(page_params.realm_message_content_edit_limit_seconds / 60);
+        }
+        return;
+    }
 
-        var message_editing_status = $("#admin-realm-message-editing-status").expectOne();
-        // grab the first alert available and use it for the status.
-        var $alerts = $(".settings-section.show .alert").hide();
-        // grab the first alert available and use it for the status.
-        var status = $("#admin-realm-notifications-stream-status");
-
-        var compose_textarea_edit_limit_minutes = $("#id_realm_message_content_edit_limit_minutes").val();
-        var new_allow_message_editing = $("#id_realm_allow_message_editing").prop("checked");
-
-        // If allow_message_editing is unchecked, message_content_edit_limit_minutes
-        // is irrelevant.  Hence if allow_message_editing is unchecked, and
-        // message_content_edit_limit_minutes is poorly formed, we set the latter to
-        // a default value to prevent the server from returning an error.
-        if (!new_allow_message_editing) {
-            if ((parseInt(compose_textarea_edit_limit_minutes, 10).toString() !==
-                 compose_textarea_edit_limit_minutes) ||
-                compose_textarea_edit_limit_minutes < 0) {
-            // Realm.DEFAULT_MESSAGE_CONTENT_EDIT_LIMIT_SECONDS / 60
-            compose_textarea_edit_limit_minutes = 10;
+    exports.extract_property_name = function (elem) {
+        return elem.attr('id').split('-').join('_').replace("id_", "");
+    };
+
+    function check_property_changed(elem) {
+        elem = $(elem);
+        var property_name = exports.extract_property_name(elem);
+        var current_val = page_params[property_name];
```

Comment:
This seems like a good place to assert that this exists.

Reply:
For settings like `realm_message_edit_limit_in_minutes`(which doesn't exist in page_params) we have to do the conversion that's why I've checked it later in else statement after checking if this refers to any valid property_name or not.

Reply:
OK, makes sense.